fhir2-appnt-api/src/main/java/org/openmrs/module/fhirappnt/api/AppointmentFhirConstants.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.fhirappnt.api;
+
+import lombok.NoArgsConstructor;
+import org.openmrs.module.fhir2.FhirConstants;
+
+@NoArgsConstructor
+public class AppointmentFhirConstants extends FhirConstants {
```

Comment:
Why do you need to extend `FhirConstants` and you already have `fhir2-ap`i as a dependency?